RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cfi.py

```diff
@@ -63,3 +63,6 @@
       chi2ThreshEE_ = cms.double(50.0),
    )                                           
 )
+
+from Configuration.StandardSequences.Eras import eras
+eras.phase2_common.toModify( ecalMultiFitUncalibRecHit.algoPSet, useLumiInfoRunHeader = cms.bool(False) )
```

Comment:
add it to the sequence you are running (at the start).
It's already in the localreco in HI and cosmics.
There is no harm from listing the same module several times in different sequences